### ISSUE TITLE
Add font size option for listbox

### ISSUE THREAD

**Issue by Robbi-Blechdose** [jMonkeyEngine-Contributions/Lemur#70]
Title. It'd be nice to be able to set the font size for the listbox contents in code just like you can for a label or a button.

**Comment by pspeed42** [comment]
Lists may not even contain text.  Only the cell renderer knows... and that's where you would set the font size.

**Comment by Robbi-Blechdose** [comment]
Right, thank you. Is there a method to set it in Java or do I need to use groovy styling?
